src/components/o-modal/_preview.html

```diff
@@ -0,0 +1,76 @@
+<!--Please create here a HTML example by using just default HTML tags-->
+
+<button onclick="openModal()">Open modal</button>
+
+<axa-modal open>
+  <h2>Prämie berechnen</h2>
+  <p><strong>Mehr als 80%</strong> unserer Kunden kombinieren die Hausratversicherung mit einer Privathaftpflichtversicherung.</p>
+  <p>Was möchten Sie?</p>
+
+  <section class="grid">
+    <div>
+      <img class="huge" src="/assets/icons/rocking_chair_blue.svg" />
+      <img class="huge" src="/assets/icons/broken_glass_blue.svg" />
+    </div>
+    <div>
+      <img class="huge" src="/assets/icons/rocking_chair_blue.svg" />
+    </div>
+    <div>
+      <axa-button data-modal-close>Hausrat und Privathaftpflicht</axa-button>
+    </div>
+    <div>
+      <axa-button data-modal-close>Nur Hausrat</axa-button>
+    </div>
+    <div>
+      <img src="/assets/icons/check_circle_outline.svg" />
+      Schützt Sie zusätzlich gegen Forderungen anderer.
+    </div>
+  </section>
+</axa-modal>
+
+<script>
+  function openModal() {
+    const modal = document.getElementsByTagName('axa-modal')[0];
+    modal.setAttribute('open', '');
+    modal.open = true;
+  }
+</script>
+
+<style>
+  h2 {
```

Comment:
even is just for demo purposes, this will destroy all other components so please add a scope

Reply:
Reproduce: open http://localhost:3000/organisms.html